doc/new-ubuntu-series.md

```diff
@@ -20,41 +20,44 @@ We maintain a branch for each ubuntu release.  The newly released branch *was*
 being built from `ubuntu/devel`, but now we want to create a `ubuntu/<release>`
 branch.
 
+```bash
     $ git clone -o upstream git@github.com:canonical/cloud-init
     $ cd cloud-init
     $ git checkout ubuntu/devel
     $ PREVIOUS_RELEASE=`dpkg-parsechangelog --offset 0 --count=1 --show-field Distribution`
     $ git checkout -b ubuntu/$PREVIOUS_RELEASE
-    $ git push upstream ubuntu/$PREVIOUS_RELEASE
 
-Note: this assumes that at this point the `ubuntu/devel` branch only
-contains commits related to the just-released Ubuntu release. If this is not
-the case prune the extra commits before pushing `ubuntu/$PREVIOUS_RELEASE`.
+    # If you DO NOT have ability to push branches as an upstream committer:
+    $ git push origin ubuntu/$PREVIOUS_RELEASE
+    # Ask for a review of your branch from cloud-init upstream committer
+    https://github.com/canonical/cloud-init/compare/ubuntu/devel...<YOUR_GITHUB_USER>:ubuntu/<PREVIOUS_RELEASE>?expand=1
```

Comment:
Does _review_ here mean to create a PR or just an informal review? In the former case, I think it would be better to compare with `main`, as by comparing with `ubuntu/devel` GH doesn't allow you to create a PR.

```suggestion
    https://github.com/canonical/cloud-init/compare/main...<YOUR_GITHUB_USER>:ubuntu/<PREVIOUS_RELEASE>?expand=1
```

Reply:
We may drop this part. Generally this is an informal review of your branch specifically, which most of us will just git diff <your_branch> upstream/ubuntu/jammy before git pushing to confirm no new diff was introduced. Since the flow to create a new upstream branch doesn't really make sense in github space. I'm wondering if generally we should just allow your user to push to create any new upstream/ubuntu/* branch to facilitate this type or work. For the one off case, I expect this is supposed the be an informal review of your remote branch and a committer would need to perform the git push upstream ubuntu/jammy from your fork.

Reply:
Thanks for the clarification!